mongodb-datastore/handlers/config.go

```diff
@@ -2,13 +2,41 @@ package handlers
 
 import (
 	keptnmongoutils "github.com/keptn/go-utils/pkg/common/mongoutils"
+	log "github.com/sirupsen/logrus"
 )
 
 const (
 	eventsCollectionName = "keptnUnmappedEvents"
 	serviceName          = "mongodb-datastore"
 )
 
+type envConfig struct {
+	// Port on which to listen for cloudevents
+	Port                    int    `envconfig:"RCV_PORT" default:"8080"`
```

Comment:
Port and Path not used, please remove them. Also most of the other env variables are not set in helm charts, please use this to set them up https://github.com/keptn/keptn/commit/bceaf4b5e298170073c566fc28361ae474e80898#diff-3cd7b0431852a6177235bd58cfc84fcbde3a836c2098a1d1bd5fab2387b8c8faR86

Example here: https://github.com/keptn/keptn/commit/bceaf4b5e298170073c566fc28361ae474e80898#diff-ded52bfd23c5a4a266c21154c749dddc94c40b6dc2c7b9fff34b455dbf87c6c5R68

Please also adapt deploy/service.yaml accordingly